x-pack/plugins/alerts/server/manual_tests/action_param_templates.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
```

Comment:
This is a handy little test script to play with changing the value of variables (the alert name) via script or ad hoc in the UI, once the script creates the actions and alerts.